packages/suite/src/hooks/settings/useEnabledNetworks.ts

```diff
@@ -16,11 +17,9 @@ export const useEnabledNetworks = (): EnabledNetworks => {
         debug: state.suite.settings.debug.showDebugMenu,
     }));
 
-    const mainnets: Network[] = NETWORKS.filter(n => !n.accountType && !n.testnet);
+    const mainnets: Network[] = getMainnets();
```

Comment:
why do you use mainnets/testnets that you are exporting in `suite-native/config/src/enabledNetworks.ts`?

Reply:
It is redundant, you are right. b4d80e61f1d3d2f38840c1417c8cb8450de8156f